packages/block-editor/src/components/global-styles/color-panel.js

```diff
@@ -231,12 +232,27 @@ function ColorPanelDropdown( {
 					};
 
 					return (
-						<Button __next40pxDefaultSize { ...toggleProps }>
-							<LabeledColorIndicators
-								indicators={ indicators }
-								label={ label }
-							/>
-						</Button>
+						<>
+							<Button { ...toggleProps } __next40pxDefaultSize>
+								<LabeledColorIndicators
+									indicators={ indicators }
+									label={ label }
+								/>
+							</Button>
+							{ hasValue() && (
+								<Button
+									__next40pxDefaultSize
+									label={ __( 'Reset' ) }
+									aria-label={
+										/* translators: %s is the type of color property, e.g., "background" */
+										__( 'Reset %s color' )
```

Comment:
This string is unfinished, it has a `%s` placeholder, but it doesn't pair with a wrapping `sprintf()` to populate that placeholder. See a proper similar usage:

https://github.com/WordPress/gutenberg/blob/86a3bc9bb3995928b940f20a934d50a1f6f80ed3/packages/components/src/tools-panel/tools-panel-header/component.tsx#L50-L54

We could actually be able to reuse the same string.